pages/settings/settings.js

```diff
@@ -500,3 +500,88 @@ proxyTypeInput.addEventListener('change', e => {
 })
 
 proxyInputs.forEach(item => item.addEventListener('change', e => setProxy(e.target.name, e.target.value)))
+
+settings.get('customBangs', (value) => {
+    const bangslist = document.getElementById('custom-bangs');
+    
+    if(value) { value.forEach(function(bang) {
+        bangslist.appendChild(createBang(bang.phrase, bang.snippet, bang.redirect))
+        bangslist.appendChild(document.createElement('br'));
```

Comment:
It'd be nicer to use some CSS to set the spacing between items rather than adding `br` elements.

Reply:
Mhm, I'll do that

Reply:
I'm no longer actually using that spacing.